stage1/modules/self-introduction/README.md

```diff
@@ -1,70 +1,69 @@
+7. Use `gh-pages` for deployment. Follow the instructions on [how to deploy an assignment from a private school repository](https://docs.rs.school/#/private-repository?id=%D0%9A%D0%B0%D0%BA-%D1%81%D0%B4%D0%B5%D0%BB%D0%B0%D1%82%D1%8C-%D0%B4%D0%B5%D0%BF%D0%BB%D0%BE%D0%B9-%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F-%D0%B8%D0%B7-%D0%BF%D1%80%D0%B8%D0%B2%D0%B0%D1%82%D0%BD%D0%BE%D0%B3%D0%BE-%D1%80%D0%B5%D0%BF%D0%BE%D0%B7%D0%B8%D1%82%D0%BE%D1%80%D0%B8%D1%8F-%D1%88%D0%BA%D0%BE%D0%BB%D1%8B).
+8. Create a Pull Request from the `self-introduction` branch to the `main` branch. The Pull Request must have the same name as the assignment – **Self-Introduction**, and it has to meet the [requirements for Pull Requests](https://docs.rs.school/#/pull-request-review-process?id=%D0%A2%D1%80%D0%B5%D0%B1%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D0%BA-pull-request-pr). As a screenshot of the result attach a screenshot of the video and provide a deployment link in the following format `https://rolling-scopes-school.github.io/your-github-account-JSFE2023Q4/self-introduction/`. **Do not merge this Pull Request.**
 
-> ### Важно
+> ### Important
 >
-> - Проверьте, что видео и текст доступны по ссылкам и для просмотра не требуется никаких дополнительных разрешений. Проще всего проверить это, открыв обе ссылки в режиме инкогнито в браузере.
-> - Мы не просим вас загружать видео в открытый доступ. Вы можете [включить доступ к ролику только по ссылке](#работа-с-youtube). Такое видео смогут просмотреть только пользователи, у которых есть нужный URL.
+> - Ensure that the video and transcript can be accessed through the links you provide and do not require any additional permissions. The easiest way to check this is to try opening both links in Incognito mode in your browser.
+> - We do not require you to make the video public. You can make it [accessible only via a link](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en). This way only the users who receive the correct URL from you will be able to watch the video.
 
-## Проверка задания
+## Task Check
 
-Факт записи видеоролика будет проверяться с помощью [кросс-чека](https://docs.rs.school/#/cross-check-flow), а его содержание будут оценивать менторы.
+During [Cross-Check](https://docs.rs.school/#/cross-check-flow) you will have to check the fact that there is a recording, but **not** its content. The content will later be assessed by mentors.
 
-Чтобы отправить выполненное задание на проверку, зайдите в [rs app](https://app.rs.school/), перейдите на вкладку Cross-Check: Submit, выберите задание **Self-Introduction** и добавьте в форму **ссылку на деплой**.
+To submit the completed task for review, go to [RS App](https://app.rs.school/), choose the Cross-Check: Submit tab, select the **Self-Introduction** task in the dropdown menu, and add your **deployment link** to the form.
```

Comment:
Maybe move this information to the workflow?

Reply:
We could probably repeat it there too as long as we have the information about an open PR to the main branch in both `Workflow` and `Task Check`.